src/main/java/org/ibp/api/java/study/StudyService.java

```diff
@@ -76,4 +79,11 @@
 	boolean hasCrossesOrSelections(final int studyId);
 
 	Integer getEnvironmentDatasetId(Integer studyId);
+
+	List<StudyEntryDto> getStudyEntries(Integer studyId, StudyEntrySearchDto.Filter filter, Pageable pageable);
```

Comment:
Aren't these new services better suited in `StudyGermplasmService` (later to be renamed to StudyEntryService) to consolidate all entries related services?